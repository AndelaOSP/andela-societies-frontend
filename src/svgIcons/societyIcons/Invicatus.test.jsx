import React from 'react';
import { shallow } from 'enzyme';

import InvictusIcon from './Invictus';

describe('<InvictusIcon />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<InvictusIcon />);
    expect(wrapper).toMatchSnapshot();
  });
});
