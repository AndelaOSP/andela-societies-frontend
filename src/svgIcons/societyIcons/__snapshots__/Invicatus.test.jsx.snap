// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InvictusIcon /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <style>
            .cls-1{fill:#fff;}
          </style>
        </defs>,
        <title>
          invictus-white-icon
        </title>,
        <path
          className="cls-1"
          d="M237.07,388.7l-41.84-49.51,35.6,5.33,0.06-.32-10.48-2.87q-14.78-4.05-29.56-8.12a2.43,2.43,0,0,1-1.16-.65q-5.84-6.81-11.61-13.67a2.1,2.1,0,0,1-.19-0.5l32.2,8.11,0.09-.26c-1-.42-2.08-0.84-3.12-1.25q-16.57-6.45-33.15-12.89a9.9,9.9,0,0,1-4.21-3.07c-3.77-4.58-7.63-9.08-11.46-13.62a7,7,0,0,1-1.31-2l8.61,3.58,76.08,31.62,45.27,18.84c1.59,0.66,3.2,1.32,4.84,1.83,1,0.3.87,0.93,0.87,1.63q0,11.29,0,22.57,0,10.52,0,21c0,0.51,0,1,0,1.65H282.54c0.29-.88.55-1.7,0.81-2.51,3.53-11.09-.55-23.61-9.79-30.07a22.56,22.56,0,0,0-30.82,5,27.59,27.59,0,0,0-5.67,17.66C237.08,386.88,237.07,387.57,237.07,388.7Z"
          transform="translate(-156.92 -293.28)"
        />,
        <path
          className="cls-1"
          d="M363.21,388.55a70.7,70.7,0,0,0-.44-7.23c-1.53-9.36-6.3-16.41-15.22-20.17a22.44,22.44,0,0,0-26.25,7.38c-6,8-7.21,16.82-4.09,26.28,0.13,0.39.24,0.78,0.4,1.32H307.55c0-.58,0-1.12,0-1.66,0-14.26,0-28.51,0-42.77a2.4,2.4,0,0,1,1.74-2.66q18.54-7.59,37-15.31,27.3-11.35,54.58-22.73L442,293.89c0.41-.17.84-0.3,1.67-0.6-2.15,2.57-4,4.82-5.91,7.06-2.47,2.93-5,5.81-7.4,8.8a9.67,9.67,0,0,1-4.24,3q-17.31,6.68-34.6,13.43a3.66,3.66,0,0,0-1.45.9l32.49-8.18c-3,3.53-6,6.63-8.47,10.13-2.57,3.64-5.92,5.33-10.14,6.38-11.18,2.79-22.26,6-33.38,9a2.32,2.32,0,0,0-1.24.69l35.6-5.33Z"
          transform="translate(-156.92 -293.28)"
        />,
        <path
          className="cls-1"
          d="M307.29,405.62H311v4.31H288.57v-4.25h4.32V344.15h-3.76v-4.3h22.42v4.24h-4.27v61.53Zm-11-61.19-0.19.06v60h0.19V344.43Zm3.91,0H300v60.1h0.18v-60.1Zm3.47,60.08h0.17V344.4h-0.17v60.09Z"
          transform="translate(-156.92 -293.28)"
        />,
        <path
          className="cls-1"
          d="M310.65,326.37l-10.56,10.12-10.3-11.42L300.38,315Z"
          transform="translate(-156.92 -293.28)"
        />,
      ],
      "data-name": "Layer 1",
      "id": "Layer_1",
      "viewBox": "0 0 286.75 116.65",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <style>
            .cls-1{fill:#fff;}
          </style>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": ".cls-1{fill:#fff;}",
          },
          "ref": null,
          "rendered": ".cls-1{fill:#fff;}",
          "type": "style",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "invictus-white-icon",
        },
        "ref": null,
        "rendered": "invictus-white-icon",
        "type": "title",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M237.07,388.7l-41.84-49.51,35.6,5.33,0.06-.32-10.48-2.87q-14.78-4.05-29.56-8.12a2.43,2.43,0,0,1-1.16-.65q-5.84-6.81-11.61-13.67a2.1,2.1,0,0,1-.19-0.5l32.2,8.11,0.09-.26c-1-.42-2.08-0.84-3.12-1.25q-16.57-6.45-33.15-12.89a9.9,9.9,0,0,1-4.21-3.07c-3.77-4.58-7.63-9.08-11.46-13.62a7,7,0,0,1-1.31-2l8.61,3.58,76.08,31.62,45.27,18.84c1.59,0.66,3.2,1.32,4.84,1.83,1,0.3.87,0.93,0.87,1.63q0,11.29,0,22.57,0,10.52,0,21c0,0.51,0,1,0,1.65H282.54c0.29-.88.55-1.7,0.81-2.51,3.53-11.09-.55-23.61-9.79-30.07a22.56,22.56,0,0,0-30.82,5,27.59,27.59,0,0,0-5.67,17.66C237.08,386.88,237.07,387.57,237.07,388.7Z",
          "transform": "translate(-156.92 -293.28)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M363.21,388.55a70.7,70.7,0,0,0-.44-7.23c-1.53-9.36-6.3-16.41-15.22-20.17a22.44,22.44,0,0,0-26.25,7.38c-6,8-7.21,16.82-4.09,26.28,0.13,0.39.24,0.78,0.4,1.32H307.55c0-.58,0-1.12,0-1.66,0-14.26,0-28.51,0-42.77a2.4,2.4,0,0,1,1.74-2.66q18.54-7.59,37-15.31,27.3-11.35,54.58-22.73L442,293.89c0.41-.17.84-0.3,1.67-0.6-2.15,2.57-4,4.82-5.91,7.06-2.47,2.93-5,5.81-7.4,8.8a9.67,9.67,0,0,1-4.24,3q-17.31,6.68-34.6,13.43a3.66,3.66,0,0,0-1.45.9l32.49-8.18c-3,3.53-6,6.63-8.47,10.13-2.57,3.64-5.92,5.33-10.14,6.38-11.18,2.79-22.26,6-33.38,9a2.32,2.32,0,0,0-1.24.69l35.6-5.33Z",
          "transform": "translate(-156.92 -293.28)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M307.29,405.62H311v4.31H288.57v-4.25h4.32V344.15h-3.76v-4.3h22.42v4.24h-4.27v61.53Zm-11-61.19-0.19.06v60h0.19V344.43Zm3.91,0H300v60.1h0.18v-60.1Zm3.47,60.08h0.17V344.4h-0.17v60.09Z",
          "transform": "translate(-156.92 -293.28)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M310.65,326.37l-10.56,10.12-10.3-11.42L300.38,315Z",
          "transform": "translate(-156.92 -293.28)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <style>
              .cls-1{fill:#fff;}
            </style>
          </defs>,
          <title>
            invictus-white-icon
          </title>,
          <path
            className="cls-1"
            d="M237.07,388.7l-41.84-49.51,35.6,5.33,0.06-.32-10.48-2.87q-14.78-4.05-29.56-8.12a2.43,2.43,0,0,1-1.16-.65q-5.84-6.81-11.61-13.67a2.1,2.1,0,0,1-.19-0.5l32.2,8.11,0.09-.26c-1-.42-2.08-0.84-3.12-1.25q-16.57-6.45-33.15-12.89a9.9,9.9,0,0,1-4.21-3.07c-3.77-4.58-7.63-9.08-11.46-13.62a7,7,0,0,1-1.31-2l8.61,3.58,76.08,31.62,45.27,18.84c1.59,0.66,3.2,1.32,4.84,1.83,1,0.3.87,0.93,0.87,1.63q0,11.29,0,22.57,0,10.52,0,21c0,0.51,0,1,0,1.65H282.54c0.29-.88.55-1.7,0.81-2.51,3.53-11.09-.55-23.61-9.79-30.07a22.56,22.56,0,0,0-30.82,5,27.59,27.59,0,0,0-5.67,17.66C237.08,386.88,237.07,387.57,237.07,388.7Z"
            transform="translate(-156.92 -293.28)"
          />,
          <path
            className="cls-1"
            d="M363.21,388.55a70.7,70.7,0,0,0-.44-7.23c-1.53-9.36-6.3-16.41-15.22-20.17a22.44,22.44,0,0,0-26.25,7.38c-6,8-7.21,16.82-4.09,26.28,0.13,0.39.24,0.78,0.4,1.32H307.55c0-.58,0-1.12,0-1.66,0-14.26,0-28.51,0-42.77a2.4,2.4,0,0,1,1.74-2.66q18.54-7.59,37-15.31,27.3-11.35,54.58-22.73L442,293.89c0.41-.17.84-0.3,1.67-0.6-2.15,2.57-4,4.82-5.91,7.06-2.47,2.93-5,5.81-7.4,8.8a9.67,9.67,0,0,1-4.24,3q-17.31,6.68-34.6,13.43a3.66,3.66,0,0,0-1.45.9l32.49-8.18c-3,3.53-6,6.63-8.47,10.13-2.57,3.64-5.92,5.33-10.14,6.38-11.18,2.79-22.26,6-33.38,9a2.32,2.32,0,0,0-1.24.69l35.6-5.33Z"
            transform="translate(-156.92 -293.28)"
          />,
          <path
            className="cls-1"
            d="M307.29,405.62H311v4.31H288.57v-4.25h4.32V344.15h-3.76v-4.3h22.42v4.24h-4.27v61.53Zm-11-61.19-0.19.06v60h0.19V344.43Zm3.91,0H300v60.1h0.18v-60.1Zm3.47,60.08h0.17V344.4h-0.17v60.09Z"
            transform="translate(-156.92 -293.28)"
          />,
          <path
            className="cls-1"
            d="M310.65,326.37l-10.56,10.12-10.3-11.42L300.38,315Z"
            transform="translate(-156.92 -293.28)"
          />,
        ],
        "data-name": "Layer 1",
        "id": "Layer_1",
        "viewBox": "0 0 286.75 116.65",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <style>
              .cls-1{fill:#fff;}
            </style>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": ".cls-1{fill:#fff;}",
            },
            "ref": null,
            "rendered": ".cls-1{fill:#fff;}",
            "type": "style",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "invictus-white-icon",
          },
          "ref": null,
          "rendered": "invictus-white-icon",
          "type": "title",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M237.07,388.7l-41.84-49.51,35.6,5.33,0.06-.32-10.48-2.87q-14.78-4.05-29.56-8.12a2.43,2.43,0,0,1-1.16-.65q-5.84-6.81-11.61-13.67a2.1,2.1,0,0,1-.19-0.5l32.2,8.11,0.09-.26c-1-.42-2.08-0.84-3.12-1.25q-16.57-6.45-33.15-12.89a9.9,9.9,0,0,1-4.21-3.07c-3.77-4.58-7.63-9.08-11.46-13.62a7,7,0,0,1-1.31-2l8.61,3.58,76.08,31.62,45.27,18.84c1.59,0.66,3.2,1.32,4.84,1.83,1,0.3.87,0.93,0.87,1.63q0,11.29,0,22.57,0,10.52,0,21c0,0.51,0,1,0,1.65H282.54c0.29-.88.55-1.7,0.81-2.51,3.53-11.09-.55-23.61-9.79-30.07a22.56,22.56,0,0,0-30.82,5,27.59,27.59,0,0,0-5.67,17.66C237.08,386.88,237.07,387.57,237.07,388.7Z",
            "transform": "translate(-156.92 -293.28)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M363.21,388.55a70.7,70.7,0,0,0-.44-7.23c-1.53-9.36-6.3-16.41-15.22-20.17a22.44,22.44,0,0,0-26.25,7.38c-6,8-7.21,16.82-4.09,26.28,0.13,0.39.24,0.78,0.4,1.32H307.55c0-.58,0-1.12,0-1.66,0-14.26,0-28.51,0-42.77a2.4,2.4,0,0,1,1.74-2.66q18.54-7.59,37-15.31,27.3-11.35,54.58-22.73L442,293.89c0.41-.17.84-0.3,1.67-0.6-2.15,2.57-4,4.82-5.91,7.06-2.47,2.93-5,5.81-7.4,8.8a9.67,9.67,0,0,1-4.24,3q-17.31,6.68-34.6,13.43a3.66,3.66,0,0,0-1.45.9l32.49-8.18c-3,3.53-6,6.63-8.47,10.13-2.57,3.64-5.92,5.33-10.14,6.38-11.18,2.79-22.26,6-33.38,9a2.32,2.32,0,0,0-1.24.69l35.6-5.33Z",
            "transform": "translate(-156.92 -293.28)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M307.29,405.62H311v4.31H288.57v-4.25h4.32V344.15h-3.76v-4.3h22.42v4.24h-4.27v61.53Zm-11-61.19-0.19.06v60h0.19V344.43Zm3.91,0H300v60.1h0.18v-60.1Zm3.47,60.08h0.17V344.4h-0.17v60.09Z",
            "transform": "translate(-156.92 -293.28)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M310.65,326.37l-10.56,10.12-10.3-11.42L300.38,315Z",
            "transform": "translate(-156.92 -293.28)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
